test/membrane_live_web/channels/event_channel_test.exs

```diff
@@ -169,7 +169,8 @@ defmodule MembraneLiveWeb.EventChannelTest do
     |> socket("event_id", %{})
     |> subscribe_and_join(MembraneLiveWeb.EventChannel, "event:#{uuid}", %{
       token: token,
-      reloaded: false
+      presenter: false,
+      requestPresenting: false
```

Comment:
Why do we need to separate this into two separate variables which are always false?

Reply:
They are only false in this test. Those values need to be separate because requesting for being presenter is a different sate than being a presenter